### DISCUSSION TITLE
how can i use nohup?

### DISCUSSION THREAD

**Discussion by fgagliazzo** [jhuckaby/Cronicle#508] [General]
i made this simple script

```
#!/bin/sh

# Enter your shell script code here
qtde=$(ps aux | grep "server.js" | grep -v grep | wc -l)

if [ $qtde -lt 1 ]
then
  nohup node /home/ubuntu/server.js > server.out &
else
  echo "server is up"
fi
```

but it keeps running until it times out in Active Jobs...

**Comment by jhuckaby** [comment]
I'm not sure why that isn't working, although I am not an expert at backgrounding processes in this fashion.

Can you try running your server in a screen session perhaps?

```
screen -dmS root "node /home/ubuntu/server.js"
```

You may have to `apt-get install screen` first.

We need a Linux expert up in here!

**Comment by mikeTWC1984** [comment]
@fgagliazzo 

I think you need to detach both stdout and stderr. Try: 
```bash
nohup node /home/ubuntu/server.js  1>server.out 2>server.out  &
```
